src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
I suggest to name the new option as `USE_RPATH_ORIGIN` for consistency with `USE_STATIC_BOOST`, `USE_SSE`, etc.  Opinions?